### PR TITLE
[Bridge/PhpUnit] doc bin/simple-phpunit

### PR DESCRIPTION
**Pull request by nicolas-grekas**
See https://github.com/symfony/symfony/pull/19915

### CODE REVIEW

File: components/phpunit_bridge.rst [hunk 1]
```diff
@@ -326,6 +329,35 @@ namespaces in the ``phpunit.xml`` file, as done for example in the
         </listeners>
     </phpunit>
 
+Modified PHPUnit script
+-----------------------
+
+.. versionadded:: 3.2
+    The modified PHPUnit script script was introduced in Symfony 3.2.
+
+This bridges provides a modified version of phpunit that you can call by using
```
Comment (javiereguiluz) [inline components/phpunit_bridge.rst]: `This bridges` -> `This bridge`

Comment (javiereguiluz) [inline components/phpunit_bridge.rst]: `of phpunit that` -> `of PHPUnit that`

File: components/phpunit_bridge.rst [hunk 2]
```diff
@@ -326,6 +329,35 @@ namespaces in the ``phpunit.xml`` file, as done for example in the
...
+  storing skipped tests on a first run, and replay them on the second run;
+* Parallelizes test suites execution when given a directory as argument, scanning
+  this directory for ``phpunit.xml.dist`` files up to ``SYMFONY_PHPUNIT_MAX_DEPTH``
+  levels (specified as an env var, defaults to ``3``);
+
+The script writes the modified PHPUnit it builds in a directory that can be
+configured by the ``SYMFONY_PHPUNIT_DIR`` env var, or in the same directory as
+the ``simple-phpunit`` if it is not provided.
+
+If you have installed the bridge through composer, you can run it by calling e.g.:
```
Comment (javiereguiluz) [inline components/phpunit_bridge.rst]: `composer` -> `Composer`

**Review by javiereguiluz — APPROVED** [review]
👍 

Status: reviewed

Thanks @nicolas-grekas!

**Comment by weaverryan** [comment]
Sweet! And I used the new PR edit feature to make the changes JAavier suggested. YES!

**Pull request merged**